charts/k6-operator/values.yaml

```diff
@@ -111,6 +111,32 @@ prometheus:
   # prometheus.enabled -- enables the prometheus metrics scraping (default: false)
   enabled: false
 
+# @schema
+# required: false
+# type: object
+# @schema
+service:
+  # @schema
+  # required: false
+  # type: boolean
+  # @schema
+  # service.enabled -- enables the k6 service (default: false)
+  enabled: false
+  # @schema
+  # additionalProperties: true
+  # required: false
+  # type: object
+  # @schema
+  # service.labels -- service custom labels
+  labels: {}
+  # @schema
+  # additionalProperties: true
+  # required: false
+  # type: object
+  # @schema
+  # service.annotations -- service custom annotations
+  annotations: {}
+
```

Comment:
Btw, can you please describe why service would need these separately defined?
I think you mentioned it in description of your use case too, that you need annotations for scrapers. Which makes sense, but why can't they be the same as for deployment?